MariaCraciun1_Task2/BoostTest.cpp

```diff
@@ -0,0 +1,75 @@
+
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MAIN
+#define BOOST_TEST_MODULE DynamicStringArrayTests
+#include <boost/test/unit_test.hpp>
+#include "DynamicStringArray.hpp"
+
+BOOST_AUTO_TEST_SUITE( DynamicStringArrayTests)
+
+  BOOST_AUTO_TEST_CASE( addEntry_getEntryTest ) 
+  {
+      DynamicStringArray obj;
+      obj.addEntry ( "Ana" );
+      obj.addEntry ( "are" );
+      obj.addEntry ( "mere" );
+      BOOST_CHECK ( obj.getEntry(2)->compare("mere") == 0 );
+  }
+
+  BOOST_AUTO_TEST_CASE ( deleteEntryTest )
+  {
+      DynamicStringArray obj;
+      obj.addEntry ( "Ana" );
+      obj.addEntry ( "are" );
+      obj.addEntry ( "mere" );
+      if(obj.deleteEntry ( "mere" ) != true && obj.getLength() !=2 )
```

Comment:
this works, but you could also use BOOST_CHECK_MESSAGE to print a nice error message